tools/pack_model_artifacts.py

```diff
@@ -0,0 +1,83 @@
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Utils to package trained model artifacts and upload to S3 bucket
+"""
+
+import os
+import argparse
+from graphstorm.config import BUILTIN_TASK_NODE_CLASSIFICATION
+from graphstorm.config import BUILTIN_TASK_NODE_REGRESSION
+from graphstorm.config import BUILTIN_TASK_EDGE_CLASSIFICATION
+from graphstorm.config import BUILTIN_TASK_EDGE_REGRESSION
+from graphstorm.inference import wrap_model_artifacts
+
+SUPPORTED_TASKS = [BUILTIN_TASK_NODE_CLASSIFICATION,
+                   BUILTIN_TASK_NODE_REGRESSION,
+                   BUILTIN_TASK_EDGE_CLASSIFICATION,
+                   BUILTIN_TASK_EDGE_REGRESSION]
+
+def main(args):
+    """ Wrap model artifacts and entry point file into a compressed tar file.
+    """
+    # chose the target entry point file according to task
+    entry_base_path = 'python/graphstorm/sagemaker/deploy/'
+    if args.gml_task == BUILTIN_TASK_NODE_CLASSIFICATION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'node_classification_entry_point.py')
+    elif args.gml_task == BUILTIN_TASK_NODE_REGRESSION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'node_regression_entry_point.py')
+    elif args.gml_task == BUILTIN_TASK_EDGE_CLASSIFICATION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'edge_classification_entry_point.py')
+    elif args.gml_task == BUILTIN_TASK_EDGE_REGRESSION:
+        path_to_entry = os.path.join(args.graphstorm_home_folder, entry_base_path,
+                                     'edge_regression_entry_point.py')
+    else:
+        raise NotImplementedError(f'Not supported GML task {args.gml_task}.')
+
+    output_file = wrap_model_artifacts(args.model_file_path,
+                                        args.model_config_yaml_paath,
+                                        args.graph_metadata_json_path,
+                                        path_to_entry, wrap_name='model',
+                                        output_path=args.output_folder)
+    print(f'Successfully pack GraphStorm model artifacts into {output_file}.')
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser('Package GraphStorm model artifacts.')
+    parser.add_argument('--model-file-path', type=str, required=True,
+                        help='The file path of trained GraphStorm model. By default, ' + \
+                             'GraphStorm saves model parameters into a binary file, named ' + \
+                             '\'model.bin\' file.')
```

Comment:
The recommendation is to a pair of parentheses for line continuation, see https://google.github.io/styleguide/pyguide.html#32-line-length and https://peps.python.org/pep-0008/#maximum-line-length

```suggestion
                        help=('The file path of trained GraphStorm model. By default, '
                             'GraphStorm saves model parameters into a binary file, named '
                             '\'model.bin\' file.'))
```